### DISCUSSION TITLE
App icon

### DISCUSSION THREAD

**Discussion by stefanocerio** [aonez/Keka#1113] [General]
I love Keka, but oh lord the icon! It keeps getting worse and worse. You definitely need a better designer!

**Comment by stefanocerio** [comment]
<img width="301" alt="Schermata 2022-06-01 alle 14 10 38" src="https://user-images.githubusercontent.com/33195873/171401288-ff6d9c92-f1e9-4e1f-aada-bcef61d1768e.png">

**Comment by gingerbeardman** [comment]
Firstly, I'd like to remind you to be respectful of others. We don't all share the same views, but that doesn't necessarily make views that don't align with your own bad or worse.

Secondly, to change the icon to one of your choosing see: https://github.com/aonez/Keka/issues/988#issuecomment-982536993

**Reply by stefanocerio** [reply]
Please do not misunderstand me , I am respectful and thankful to all the team behind keka, and as I said I love the app.
But, in all the many years that I have been recommending your app, I haven’t found one person that liked the icon.
I would definitely invest some more money in redesigning it from the ground up.
Why don’t you start asking your users if they like it or not.

**Reply by gingerbeardman** [reply]
You know the icon has changed over the years? The current "squarcle" version was introduced quite recently, around 6 months ago, in November 2021. Before that there was the round one for many years. Which one is it that you don't like?

BTW both of these icons were drawn by world famous (no exaggeration) icon designer David Lanham and I can't imagine he designs icons for free :)

I'm not the developer of the app, just a user and contributor, but my opinion of the icon is irrelevant. It's not our decision. If I don't like an app icon, I just assign a custom icon and 10 seconds later I can carry on with my life. I kindly suggest you should do the same.

**Reply by stefanocerio** [reply]
If your are not among the developers , I do not understand why you are talking. I assure you I am able to carry on with my life even with keka's icon in my dock🤣🤣 
I've been changing the icon every time after every update for years , thanks for your suggestion. 
None of the Icons are any good, the last one , I say again, is even worse.

**Reply by gingerbeardman** [reply]
I am not the developer of the app, I am a contributor to the app. I'm sure you can look up the definitions and figure out the differences. You can also check the Keka about window. Take care!

**Comment by gingerbeardman** [comment]
<img width="92" alt="171460961-382b9eab-0420-44d2-bba3-629341041215" src="https://user-images.githubusercontent.com/49612/171461060-3806b790-943d-4ef8-bb81-0a1708588728.png">

**Reply by aonez** [reply]
Long time no see a vertical Dock 😅

**Comment by aonez** [comment]
@stefanocerio it's hard to make an icon that everyone likes. I've received mostly positive feedback, but obviously also bad feedback like yours. I like all the Keka icons, obviously, and David Lanham is an awesome designer that I had the luck to work with in the last two iterations.

The ideal solution for the different likes of everyone will be to adopt the [setAlternateIconName](https://developer.apple.com/documentation/uikit/uiapplication/2806818-setalternateiconname) iOS API in the macOS framework. My suggestion is always to [send feedback](https://www.apple.com/feedback/) to Apple about this, the more feedback they get the changes to adopt this feature.
